### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 12th of July 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#797] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 12th of July 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

* Zoom link:  https://us06web.zoom.us/j/84336116610
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: [link](https://www.youtube.com/watch?v=r3VPy5tsKmE&ab_channel=Kinvolk)
- Community Calls calendar (all future calls): [link]()
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- future events: come and say bonjour to @tormath1 if you are in Lyon! https://github.com/sre-france/meetups/issues/52
- past events: we were last week at KCD Berlin, recordings will be available soon! 
- latest bug smashing day on Friday: https://hackmd.io/eYCdnFnfTyO3hUaxtse5wQ)

## Spotlights
- @JAORMX from Equinix Metal will tell the latest and the greatest regarding SELinux https://github.com/flatcar-linux/Flatcar/issues/673

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by tormath1** [comment]
Hi - I could briefly mention the talk I'll gave about Flatcar / Ignition in a Meetup in Lyon :fr: (https://github.com/sre-france/meetups/issues/52)

**Comment by JAORMX** [comment]
@tormath1 and I could speak about the work that's been happening around SELinux. We're hoping to close https://github.com/flatcar-linux/Flatcar/issues/673 in the near future.

**Comment by tormath1** [comment]
@miao0miao we can also mention in the news the latest bug smashing day (here's some notes produced during the event: https://hackmd.io/eYCdnFnfTyO3hUaxtse5wQ)

**Comment by miao0miao** [comment]
we mentioned in the meeting the "first good issue" tag. if you feel like contributing too, you can have a look here:

https://github.com/flatcar-linux/Flatcar/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Fdocs+label%3A%22good+first+issue%22
